satellite/satellitedb/utils_test.go

```diff
@@ -47,3 +47,18 @@ func TestPostgresNodeIDsArray(t *testing.T) {
 
 	assert.Equal(t, expected.(string), string(got.([]byte)))
 }
+
+func TestPostgresPathsArray(t *testing.T) {
+	ids := make([][]byte, 10)
+	for i := range ids {
+		ids[i] = testrand.BytesInt(testrand.Intn(1000))
```

Comment:
Do you think this is better than using `testrand.NodeID().Bytes()`?

Reply:
Reverted